packages/gallery/src/components/gallery/proGallery/galleryContainerExtraNew.js

```diff
@@ -48,7 +48,7 @@ export class GalleryContainer extends React.Component {
         this.initialGalleryState = galleryState;
       }
     } catch (_e) {
-      console.warn(_e);
+      console.warn('Cannot create initial state from props (blueprints)', _e);
```

Comment:
Thanks, I dont want the view components to have the word blueprint. the layoutingIndex should be the only one and it is {...blueprint} to the core.